woocommerce/woocommerce-android
# Updates Colored Buttons Style used in Login flow

Closes: #9165 


### Description

The PR updates Colored Buttons to make it consistent with the rest of the app. Font color in both modes on these buttons should be white.

p1684140904749489/1683630224.493639-slack-C0354HSNUJH

### Testing instructions

* Run login flow in both dark and light mode
* Notice that the buttons have white font color, the same as the buttons inside of the app

### Images/gif

<img width="203" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/f22b4396-8cf0-4da5-b375-a47a60442491">
<img width="203" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/e9a75d6e-cef9-47b2-88a3-199472a61620">
